Holovibes/includes/gui/selections/overlay_manager.hh

```diff
@@ -14,7 +14,7 @@ namespace holovibes::gui
 {
 /*! \class OverlayManager
  *
- * \brief #TODO Add a description for this class
+ * \brief class that manages the overlays of a window
```

Comment:
**C**lass
If it was me I would permute this description and the file's description, so that the file one would be a general sentence and the class doc would be a bit more detailed
What do you think ? 
(I guess this applies to every other class)